format/tap/testdata/basic_prog1.fqtest

```diff
@@ -0,0 +1,21 @@
+$ fq -d tap dv basic_prog1.tap
```

Comment:
Possible to add some instruction how the tap and tzx test files were created? if very simple just a comment in the .fqtest-file otherwise maybe a README.md in testdata?